packages/module/src/ChatbotConversationHistoryNav/ChatbotConversationHistoryNav.scss

```diff
@@ -230,4 +229,14 @@
     --pf-v6-c-drawer__head--PaddingInlineStart: var(--pf-t--global--spacer--lg);
     --pf-v6-c-drawer__head--PaddingInlineEnd: var(--pf-t--global--spacer--lg);
   }
+  .pf-v6-c-drawer__close {
```

Comment:
Are these changes properly scoped to being just for the compact variant?

Reply:
Yes! They are tied to `.pf-chatbot__history.pf-m-compact {` on line 208.